packages/block-library/src/buttons/deprecated.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import { omit } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { InnerBlocks } from '@wordpress/block-editor';
+
+const deprecated = [
```

Comment:
There are integration tests for all blocks that add for deprecated versions, too. It helps to ensure that legacy version of blocks continue to work as new changes are introduced. See: https://github.com/WordPress/gutenberg/blob/master/docs/contributors/testing-overview.md#core-block-testing

Reply:
Thanks. I've added the fixtures now. :+1: